src/components/va-alert/va-alert.tsx

```diff
@@ -0,0 +1,117 @@
+import { Component, Event, EventEmitter, Host, Prop, h } from '@stencil/core';
+
+@Component({
+  tag: 'va-alert',
+  styleUrl: 'va-alert.css',
+  shadow: true,
+})
+export class VaAlert {
+  /* Determines the icon and border/background color.
+   * One of `info`, `error`, `success`, `warning`, or `continue`
+   */
+  @Prop() status: string = 'info';
+
+  /**
+   * If true, renders the alert with only a background color corresponding
+   * to the status - no icon or left border.
+   */
+  @Prop() backgroundOnly: boolean = false;
+
+  /**
+   * If true, doesn't fire the CustomEvent which can be used for analytics tracking
+   */
+  @Prop() disableAnalytics: boolean = false;
+
+  @Prop() visible: boolean = true;
+
+  /**
+   * Headline for the alert. Won't show if `backgroundOnly` is true
+   */
+  @Prop() headline: string;
+
+  /**
+   * Determines the level of the heading in the alert
+   */
+  @Prop() level: number = 3;
+
+  @Prop() closeBtnAriaLabel: string = 'Close notification';
+
+  @Prop() closeable: boolean = false;
+
```

Comment:
You thinking this scroll stuff is unnecessary?  I only see one current component (in _ContactInformationActionButtons.jsx_) that deviates from the default value.

```js
  /**
   * Defaults to 'start' but customizable.
   */
  scrollPosition: PropTypes.string,
```

Reply:
I left it out because of the problematic smooth scroll behavior in the `CollapsiblePanel` component.

Reply:
Here are some links to scrolling problems in other components:

- https://dsva.slack.com/archives/C01FNSTQD7H/p1617036059425700?thread_ts=1617024286.386200&cid=C01FNSTQD7H
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/6461

My understanding is that it increases cognitive load, and also causes problems for users with motion sensitivity.

Reply:
Thanks for the links!

I know I hated the scrolling myself. It's an unexpected behavior and I didn't want it.